torch/_inductor/codecache.py

```diff
@@ -2166,9 +2166,9 @@ def load(
         extra: str = "",
         linemap: Optional[List[Tuple[int, str]]] = None,
         attrs: Optional[Dict[str, Any]] = None,
-    ) -> ModuleType:
+    ) -> Tuple[ModuleType, str]:
         key, path = write(source_code, "py", extra=extra)
-        return cls.load_by_key_path(key, path, linemap, attrs)
+        return cls.load_by_key_path(key, path, linemap, attrs), path
```

Comment:
will this have any interaction with cached triton kernels (Maybe not) ? What's the path there ? cc @oulgen

Reply:
Path is the python kernel file inductor saved in /tmp/username_inductor/..., it is a hash_code.py file.